src/runtime/pay-flow-test.js

```diff
@@ -426,6 +427,49 @@ describes.realWin('PayCompleteFlow', {}, env => {
           publicationId: 'pub1',
           loginHint: 'test@example.org',
           productType: ProductType.SUBSCRIPTION,
+          isSubscriptionUpdate: false,
+        }
+      )
+      .returns(Promise.resolve(port));
+    return flow.start(response);
+  });
+
+  it('should have valid flow constructed w/ oldSku', () => {
+    // TODO(dvoytenko, #400): cleanup once entitlements is launched everywhere.
+    const purchaseData = new PurchaseData();
+    const userData = new UserData('ID_TOK', {
+      'email': 'test@example.org',
+    });
+    const response = new SubscribeResponse(
+      'RaW',
+      purchaseData,
+      userData,
+      null,
+      ProductType.SUBSCRIPTION,
+      null,
+      'oldSku'
```

Comment:
use a different string to represent the old sku. It's confusing to read this and differentiate from the name of the object key.

Reply:
Done!